services/challengedeployerservice/controller.go

```diff
@@ -18,10 +12,81 @@ import (
 	"github.com/sdslabs/katana/lib/utils"
 	"github.com/sdslabs/katana/types"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"log"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
 )
 
-func DeployChallenge(c *fiber.Ctx) error {
+func Deploy(c *fiber.Ctx) error {
```

Comment:
Why is this function changed?

Reply:
Deploy is for get request.
DeployChallenge is for post request. 
Was testing something. Now restored